app/src/main/java/hm/orz/chaos114/android/androidsaddlersample/MainActivity3.java

```diff
@@ -0,0 +1,40 @@
+package hm.orz.chaos114.android.androidsaddlersample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+
+public class MainActivity3 extends AppCompatActivity {
+
+    private boolean FOO;
+    private boolean BAR;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName